cloud

```diff
@@ -1752,6 +1756,7 @@ function execute_command() {
                           smtp_mocked \
                           test \
                           log_level \
+                          cache_bucket_name \
```

Comment:
We don't need to pass this parameter to `execute_command()`, because it is supposed to generate it itself, from the project name and the prefix.

Reply:
done